README.md

```diff
@@ -69,7 +69,7 @@ Make sure your users know what stage a feature is by adding badges in the help t
     #' \Sexpr[results=rd, stage=render]{lifecycle::badge("experimental")}
```

Comment:
I think we should try defining a [package macro](https://cran.rstudio.com/doc/manuals/r-devel/R-exts.html#User_002ddefined-macros), so all you need is `\lifecycle{experimental}`.

It's worth trying to see if we can link to images in another package (probably not)

Reply:
I remember trying both of these a while back. I'm pretty sure we can't use images from another package. I may have given up the macro thing because of R 3.1 support, I'll try again.